src/utils/getVaults.js

```diff
@@ -2,11 +2,11 @@ const axios = require('axios');
 
 const getVaults = async vaultsEndpoint => {
   try {
-    const response = await axios.get(vaultsEndpoint);
-    const data = response.data;
-    let vaults = '[' + data.substring(data.indexOf('\n') + 1);
-    vaults = eval(vaults);
-    return vaults;
+    // const response = await axios.get(vaultsEndpoint);
```

Comment:
What's the purpose of this change?

Reply:
ts supposed to pull the data from the github link like this:

https://raw.githubusercontent.com/beefyfinance/beefy-app/prod/src/features/configure/vault/moonbeam_pools.js

so once that code is in the app part and its public, we can include the url and it'll pull the data from there. currently i'm pulling it from a hardcoded version of that in the api repo but ideally it'd be like above so i was saving that code for when we switch back